src/api/app/views/webui2/webui/request/_superseded_by_message.html.haml

```diff
@@ -0,0 +1,24 @@
+- if superseding.present? && superseded_by.nil? && diff_to_superseded.nil?
+  .alert.alert-secondary{ role: 'alert' }
+    %i.fa.fa-info-circle.text-info
+    This request supersedes:
+    - superseding.each do |bs_request|
+      = link_to("request #{bs_request.number}", request_show_path(number: bs_request.number))
+      = surround('(', ")#{', ' unless bs_request == superseding.last}") do
```

Comment:
```
 = surround('(', ")#{', ' unless bs_request == superseding.last}") do
```
there isn't a better way to do this? for me looks like cryptic

Reply:
I tried also with doing something like this:

```ruby
= surround('(', ')') do
  = link_to('Show diff', request_show_path(number: bs_request.superseded_by, diff_to_superseded: bs_request.number))
= ', ' unless bs_request == superseding.last
```

This would be a lot more readable, but there is then a space between the closing parenthesis and the comma.

Reply:
maybe a helper @dmarcoux ?

Reply:
I have the feeling that it is super specific and that it would be used only once.

Reply:
I'm with @dmarcoux, this is something very specific

Reply:
I don't think the helpers are just about how many times it is being used, but as well about simplify the view logic. what we have now ```= surround('(', ")#{', ' unless bs_request == superseding.last}")``` is really easy to oversee while working with the view but at the same time hard to understand what's going on.